README.md

```diff
@@ -160,6 +160,18 @@ Example:
 - Twitter: [@twitterhandle](https://twitter.com/AmreNurgul)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/amre-nurgul/)
 
+👤 **Sajeel Zafar**
+
+- GitHub: [@Sajeelzafar](https://github.com/Sajeelzafar)
+- Twitter: [@SajeelZafar6](https://twitter.com/SajeelZafar6)
+- LinkedIn: [Sajeel Zafar](https://www.linkedin.com/in/sajeelzafar/)
+
+👤 **Tashobya Israel**
+
+- GitHub: [@githubhandle](https://github.com/tashisrael)
+- Twitter: [@twitterhandle](https://twitter.com/tashisrael)
+- Linkedin: [@linkedinhandle](https://www.linkedin.com/in/tashobya-israel-6a66b0181/l)
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Great job working on this project. On line 239 of your ReadMe file is the MIT license link, but when I click on the link, I get erorr 404. Please kindly kix this as it is not professional to have broken links in our ReadMe
![Screen Shot 2023-02-07 at 12 39 13 AM](https://user-images.githubusercontent.com/62849872/217194366-64e3121b-fae8-4dea-aa86-bcbcbdc56eb3.png)